projects/plugins/wpcomsh/README.md

```diff
@@ -106,8 +106,8 @@ In order for WordPress to load the test site correctly, you'll need to access th
 - Browse to http://nginx:8989 in your favorite web client / browser
 
 ## Deployment
-
-- Use the Monorepo deployment tools available in the `tools` folder in the root. For more information on releasing a plugin go to [the Monorepo deployment docs](../../../docs/monorepo.md#plugin-release-tooling).
+- After PR is merged, the CI will automatically create a rolling release for you. You can find [automatic release here](https://github.com/Automattic/wpcom-site-helper/releases). The atomic infrastructure monitor releases on this page and pick up the new release and deploy it to all the sites.
+- If you are in hurry and need to deploy urgent release, you can do it manually by creating a new release on GitHub and the atomic infrastructure will pick it up and deploy it to all the sites. To create manual release, You need to use the Monorepo deployment tools available in the `tools` folder in the root. For more information on releasing a plugin go to [the Monorepo deployment docs](../../../docs/monorepo.md#plugin-release-tooling).
```

Comment:
I am not if this is entirely correct.

Reply:
I found this FG which might be relevant: PCYsg-Osp-p2

Reply:
Thanks.